### ISSUE TITLE
DEV: LOG: AddToStore JUnit Testing

### ISSUE THREAD

**Issue by TessForGithub2** [hashmapinc/Drillflow#402]
Size:
Large (4 point)

This card will verify that AddToStore was successfully implemented.

Included in this card, in addition to the JUnit Testing, will be verification that the mappings happened successfully. Where appropriate, TODO markers will also be placed in the code of UOM work (may as well put this in while reviewing the mapping functionality).

**Comment by TessForGithub2** [comment]
FOCUS for 4/30
*****************

Working on JUnit for DotDelegator for log. The idea is:
1. unmarshal incoming JSON String that represents channelSet + channels as a response from DoT
2. now the POJOs are populated from the unmarshal operation, so perform the transformations per mapping document to return a JSONObject ready for transformation to WITSML XML (performed by the caller before returning as XMLOUT)

I have completed the unmarshal; I will test once Sukhe puts in the fix I requested last night so he is sending me more than channels (a JSONArray) but a JSONObject as described above.

I will continue on with mapping & writing JUnit testing against this module.

**Comment by niaalex** [comment]
Data is now unmarshalled successfully; subset of transformations entered to test E2E and to unblock QA.

**Comment by niaalex** [comment]
On target for 5/1 release to testing

**Comment by niaalex** [comment]
Per @TessForGithub2 ; this item is pending feedback from @cherrera2001 to inquiry care of (c/o)   "mock of the REST calls is successful for the 1st call but failing with a null pointer on the 2nd call; Theresa is following a rough guideline in StackOverflow" (link was provided in Slack to Chris) FYI

Update pending response from Chris. 

New ETA is 6/17 by 3pm call

**Comment by niaalex** [comment]
@TessForGithub2  please advise if this is still open

**Comment by TessForGithub2** [comment]
UPDATE: Chris contributed and stream-lined my old mocking. I have included his stream-lined mocking in the code. However, there is another issue where the data URL is null when the test is run. When I force a value for this, it works (FINALLY!).

So just one more issue and this will be closed.

**Comment by TessForGithub2** [comment]
Update: Tested and working.

**Comment by TessForGithub2** [comment]
UPDATE: Will create my own branch to release now.
